crates/op-consensus/src/receipt/mod.rs

```diff
@@ -0,0 +1,199 @@
+use alloy_primitives::{Bloom, Log};
+
+mod envelope;
+pub use envelope::OpReceiptEnvelope;
+
+mod receipts;
+pub use receipts::{OpReceipt, OpReceiptWithBloom};
+
+/// Receipt is the result of a transaction execution.
+pub trait OpTxReceipt {
```

Comment:
we already have this in alloy?
minues the deposit functions?

so we could unify the traits and inherit here?

Reply:
Yeah good thinking. Done.